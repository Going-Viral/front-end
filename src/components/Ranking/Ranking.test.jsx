import React from 'react';
import { shallow } from 'enzyme';
import Ranking from './Ranking';
import { ThemeProvider } from '@material-ui/core';

describe('Ranking component', () => {
  it('renders Ranking', () => {
    const wrapper = shallow(
      <ThemeProvider>
        <Ranking />
      </ThemeProvider>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
