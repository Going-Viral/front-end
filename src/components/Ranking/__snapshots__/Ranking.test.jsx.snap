// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ranking component renders Ranking 1`] = `
<ContextProvider>
  <Component />
</ContextProvider>
`;
